### PR TITLE
Top-navigation implementation

### PR DESCRIPTION
**Pull request by DarinaBlazevica**

### CODE REVIEW

**Review by ilze-dom — APPROVED** [review]

File: src/components/molecules/Menu/Menu.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import "./Menu.scss";
+
+const Menu = ({menu, showMenu}) => {   
+  return (
+    <div className={showMenu}>
+      <a href={menu.url}>{menu[0].title}
```
Comment (ReinisZogots) [inline src/components/molecules/Menu/Menu.jsx]: Use map, do not hard-code every menu item

File: src/components/organisms/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -37,21 +37,19 @@ const Carousel = (props) => {
 
   return (
     <div className="carousel-container">
```
Comment (ReinisZogots) [inline src/components/organisms/Carousel/Carousel.jsx]: maybe name this one "carousel"

File: src/structure.json [hunk 3]
```diff
@@ -1,7 +1,15 @@
 {
+  "topnav" :
```
Comment (ReinisZogots) [inline src/structure.json:2]: align indentation of topnav  and menu with other content

File: src/components/molecules/SubNavigation/SubNavigation.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <a className="top-nav__sub-navigation--back" id="sub-nav-back" href="#">
+          <button className="top-nav__sub-navigation--backbtn" id="btn__back">
+            <img
+              src="./svg_icons/angle-left-solid-black.svg"
+              alt="back button"
+            />
+          </button>
+          <p>Back</p>
+        </a>
+        <a href="#">One Page</a>
```
Comment (ReinisZogots) [inline src/components/molecules/SubNavigation/SubNavigation.jsx:17]: use map

File: src/components/molecules/Menu/Menu.jsx (hunk 1 above)
Comment (DarinaBlazevica) [reply src/components/molecules/Menu/Menu.jsx]: Changed

File: src/components/organisms/Carousel/Carousel.jsx (hunk 2 above)
Comment (DarinaBlazevica) [reply src/components/organisms/Carousel/Carousel.jsx]: Changed

File: src/components/molecules/SubNavigation/SubNavigation.jsx (hunk 4 above)
Comment (DarinaBlazevica) [reply src/components/molecules/SubNavigation/SubNavigation.jsx:17]: Will be in another PR for Subnavigation

File: src/structure.json (hunk 3 above)
Comment (DarinaBlazevica) [reply src/structure.json:2]: This is a little confusing, what exactly is not correct there  ?

Comment (DarinaBlazevica) [reply src/structure.json:2]: changed

**Review by Jalal999 — APPROVED** [review]

**Pull request merged**